### PR TITLE
CH-8723: save timestamp fields as Date objects with a UTC time zone, …

### PR DESCRIPTION
**Pull request by jonbasebl5**
…so it's actually correct

### CODE REVIEW

**Comment by mrmorris** [comment]
+1

**Comment by liammccartney** [comment]
Whoa - this repo is on Github!? 😍

File: src/Adapter/Zend.php [hunk 1]
```diff
@@ -314,7 +315,9 @@ private function _map($data, Entity_Abstract $entity) {
             }
 
             // initialize date types
-            if (in_array($type, ['datetime', 'timestamp', 'date', 'time'])) {
+            if ($type == 'timestamp') {
+                $value = empty($value) ? null : new CourseHorse_Date($value, new DateTimeZone('UTC'));
+            } elseif (in_array($type, ['datetime', 'date', 'time'])) {
```
Comment (liammccartney) [inline src/Adapter/Zend.php]: It seems that this `elseif` isn't strictly necessary, and also doesn't fit the pattern that this function seems to follow. But I don't feel strongly about that.

**Review by liammccartney — APPROVED** [review]
Sweet!

File: src/Adapter/Zend.php (hunk 1 above)
Comment (jonbasebl5) [reply src/Adapter/Zend.php]: what do you mean re: it not being necessary?

I see what you mean re: pattern, I'm down to change it

Comment (liammccartney) [reply src/Adapter/Zend.php]: Apologies, was unclear:
```php
if ($type == 'timestamp') {
   $value = empty($value) ? null : new CourseHorse_Date($value, new DateTimeZone('UTC'));
} 
if (in_array($type, ['datetime', 'date', 'time'])) {
   // ...
}
```
achieves the same behavior

**Pull request merged**